gramps/gen/plug/_pluginreg.py

```diff
@@ -362,6 +364,14 @@ class PluginData:
     .. attribute:: reset_system
        Boolean to indicate that the system (sys.modules) should
        be reset.
+
+    Attributes for RULE plugins
+
+    .. attribute:: ruleclass
+       The class (Person, Event, Media, etc.) the rule applies to.
+
+       Note that the rule plugin id MUST be the exact class name of the rule;
```

Comment:
Use the `ruleclass` attribute to hold the class of the rule.  Don't use the `id` for this.

Add a `namespace` attribute to hold the class that the rule applies to.  I see that the term namespace is used for this elsewhere in the rules.

Reply:
Changes done as you suggest...